tests/login.test.js

```diff
@@ -0,0 +1,52 @@
+import LOGIN_PAGE from '../pages/login.page';
```

Comment:
Convention naming for page objects in webdriver.io favours **PascalCase** for page objects. Make sure to name this accordingly (LoginPage instead of LOGIN_PAGE, which is a pattern used for constants)

Reply:
Thank you! When I used Replace All to change the calls to consts ,I didnt notice that everything had changed